src/shogun/lib/config.h.in

```diff
@@ -41,6 +41,8 @@
 #cmakedefine USE_LOGCACHE 1
 #cmakedefine USE_LOGSUMARRAY 1
 
+#cmakedefine JINJA2_FOUND "@JINJA2_FOUND@"
```

Comment:
please try to be consequent about naming variables: `USE_*` or `HAVE_*`... and why would you need this macro in C?